Code completion in completionvalues1.xml for <item name="android:shadowColor">@an^</item> selecting @android::
@@ -6 +6
-         <item name="android:shadowColor">@an^</item>
+         <item name="android:shadowColor">@android:^</item>
